### ISSUE TITLE
about the BUS Dataset B experiment

### ISSUE THREAD

**Issue by Liuyu04** [nabsabraham/focal-tversky-unet#5]
Dear Nabila Abraham:
      I have a question about your paper,  when i run your codes, the ISIC dataset result can be able to reach the scope of the paper dsc: 0.849 ~ 0.863, but the bus dataset B result is uncertain and can't achieve the results in the paper dsc: 0.78 ~ 0.839. Do you konw why?  
     the paper mention the experiment is 5-fold cross-validation, it says your hyperparameters got through this?

**Comment by nabsabraham** [comment]
Hi Liu! I hope my email replies were sufficient for you - as I said, the FTL is a polynomial loss function, there are sometimes stability issues that I mitigated using deep supervision (as an averaging method) and early stopping and model checkpointing (to get the best model as the loss convergence was not always stable) and parameter initialization (used in this repo). Hope this helps any future research directions! :)

**Comment by Halleyawoo** [comment]
> Dear Nabila Abraham:
> I have a question about your paper, when i run your codes, the ISIC dataset result can be able to reach the scope of the paper dsc: 0.849 ~ 0.863, but the bus dataset B result is uncertain and can't achieve the results in the paper dsc: 0.78 ~ 0.839. Do you konw why?
> the paper mention the experiment is 5-fold cross-validation, it says your hyperparameters got through this?

HI，could you share the link of these two dataset？
